assets/src/scss/je-docs.scss

```diff
@@ -21,3 +21,8 @@
 @import 'components/icons';
 @import 'components/list--tiled';
 @import 'components/logo';
+
+.c-footer {
+    // Temporary fix to stop the apps row from overflowing the demo block in mobile view
```

Comment:
Not sure I like to see TODO comments (IMHO "Temporary fix..." implies TODO)! What work is required for a permanent fix?

Reply:
@simonsandersje Short answer: I don't know.

Longer answer: The problem is that the footer row containing the app download links is wider than the footer. This is only visible within the demo block and only for mobile view.

<img src="https://user-images.githubusercontent.com/26894168/39197611-cd88f26c-47dc-11e8-9a38-8cf426c180a2.png" width=250px>

Hiding the overflow also hides the overflow for the header demos.
<img src="https://user-images.githubusercontent.com/26894168/39197842-78868620-47dd-11e8-994d-4931c47a8cb2.png" width=400px>

Using `overflow-x: hidden` doesn't work, as this is incompatible with `overflow-y: visible` (it's a [problem with the spec](https://stackoverflow.com/a/6433475)), it computes as `overflow-y: scroll` which makes the header demo look naff *and* doesn't actually show the overflow. 😞 

If you've got any suggestions they'd be greatly appreciated!

Reply:
Fixing in [this PR](https://github.com/justeat/f-footer/pull/28).